app/services/ccms/attribute_value_generator.rb

```diff
@@ -302,11 +302,15 @@ def benefit_check_passed?(_options)
 
     def proceeding_cost_limitation(options)
       proceeding = options[:proceeding]
-      return "MULTIPLE" if proceeding.used_delegated_functions?
+      return "MULTIPLE" if proceeding.used_delegated_functions? || multiple_scope_limitations?(proceeding)
 
       proceeding.substantive_scope_limitation_code
     end
 
+    def multiple_scope_limitations?(proceeding)
+      proceeding.scope_limitations.where(scope_type: :emergency).count > 1 || proceeding.scope_limitations.where(scope_type: :substantive).count > 1
+    end
```

Comment:
If you only want to query the database once, I think you should be able to do something like this

```ruby
def multiple_scope_limitations?(proceeding)
  proceeding
    .scope_limitations
    .select(:scope_type)
    .group(:scope_type)
    .count
    .any? { |type, count| count > 1 }
end
```

Reply:
Thanks! have amended the query.